nab/detectors/htmcore/README.md

```diff
@@ -0,0 +1,115 @@
+# Numenta and NumentaTM detectors
+
+This directory holds the Python 2 code required to run the `numenta` and
+`numentaTM` detectors against the NAB data. In 2019 the main body of the
```

Comment:
TODO readme needs to be updated for htm.core, or probably just remove as it's a normal py3 detector and just link Readme to htm.core/Readme

Reply:
updated